### DISCUSSION TITLE
local media is double escaped using the CLI

### DISCUSSION THREAD

**Discussion by Gagi2k** [LMS-Community/slimserver#968] [General]
Using the CLI to query the local media like this:

`tracks 0 50 album_id:2541 tags:AuKjJtc`

returns the following response:

`tracks 0 50 album_id%3A2541 tags%3AuKjJtc id%3A30137 title%3AA%20Beautiful%20Lie url%3Afile%3A%2F%2F%2Fhome%2Fmusic%2Fmusic_files%2FNAS%2FMusik%2520sortiert%2FAlternative%2C%2520Indie%2C%2520Hardcore%2F30%2520Seconds%2520to%2520Mars%2F2006%2520-%2520A%2520Beautiful%2520Lie%2F30%2520Seconds%2520to%2520Mars%2520-%25203%2520-%2520A%2520Beautiful%2520Lie.mp3 coverart%3A1 artwork_track_id%3Acc68494c tracknum%3A3 coverid%3Acc68494c`

The url is double escaped. The unescaped version of the string would be this:

`url:file:///home/music/music_files/NAS/Musik%20sortiert/Alternative,%20Indie,%20Hardcore/30%20Seconds%20to%20Mars/2006%20-%20A%20Beautiful%20Lie/30%20Seconds%20to%20Mars%20-%203%20-%20A%20Beautiful%20Lie.mp3
`

while the correct path contains spaces and is this:

`url:file:///home/music/music_files/NAS/Musik sortiert/Alternative, Indie, Hardcore/30 Seconds to Mars/2006 - A Beautiful Lie/30 Seconds to Mars - 3 - A Beautiful Lie.mp3`

**Comment by michaelherger** [comment]
The URL must be escaped. And the overall CLI response is escaped once again. So this is correct. It's a URL, not the path as existing on your disk.

**Reply by Gagi2k** [reply]
ahh right, sorry for the noise!
